.github/workflows/tests.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       - uses: r-lib/actions/setup-r@v2
       - uses: r-lib/actions/setup-r-dependencies@v2
         with: 
-          extra-packages: digest, RJSONIO, gtable, plyr, reshape2, scales, knitr
+          extra-packages: digest, RJSONIO, gtable, plyr, reshape2, scales, knitr, chromote
```

Comment:
can you please delete extra-packages here? 
It is redundant with DESCRIPTION
and https://github.com/r-lib/actions/tree/v2/setup-r-dependencies says that "By default all direct dependencies of the current package are installed" so extra-packages should not be necessary.

Reply:
done